Sources/ATTNAPI.m

```diff
@@ -483,7 +483,7 @@ - (NSMutableDictionary*)buildBaseMetadata:(ATTNUserIdentity*)userIdentity {
 
 + (NSString* _Nullable)extractDomainFromTag:(NSString*)tag {
   NSError* error = nil;
-  NSRegularExpression* regex = [NSRegularExpression regularExpressionWithPattern:@"window.__attentive_domain='(.*?).attn.tv'" options:0 error:&error];
+  NSRegularExpression* regex = [NSRegularExpression regularExpressionWithPattern:@"='([a-z0-9-]+)[.]attn[.]tv'" options:0 error:&error];
```

Comment:
will this work with dev companies that have, for example, `siren.dev.attn.tv`?

Reply:
It won't - that was a decision made early in the process afaik to avoid unnecessary code complexity on a public repo. We have a prod test company for that reason, AttentiveGames